index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+<meta chaarster="utf-8">
+<meta name="viewport" content="with=device-with, initial-scale=1">
+<title>My First Website</title>
+    </head>
+    <body>
+        <h1>PERSONAL STATEMENT </h1>
+        <h2>Materials Science and Engineering</h2>
+<p>
+    "Engineers belong to one of the greatest professions in the world, and are responsible for many of the human race’s greatest technological achievements. Well that’s my view anyway and it’s partly for this reason that I aspire to become a qualified, professional and successful engineer, specializing in the microstructural and mechanical properties of materials. In constantly changing environments ranging from space travel to automobile manufacture, the role of the engineer is crucial. I have a strong desire to be involved in these exciting industries and to help come up with practical solutions to the challenges that the various different communities face.
+
+
+
+</p>
+<p>
+    <font color="blue"><to me, engineering is an interesting field and absorbing subjec</font> There is always something new to learn. I’m attracted by its dynamic environment in which new materials, technologies and processes are continually being developed. As a hands on person, I am driven to work in a laboratory where I will be able to come up with ideas and results, and then be involved in developing and implementing them. For me personally there is a lot of satisfaction to be gained from challenging yourself and then achieving.
+    
+     </p>
+     <p> <u>This is really what I have done and want to continue doing.</u> In my master thesis I talked about the Microstructural and Mechanical Properties of a platelet (W,M1,M2)C-Co <a href="https://www.linkedin.com">Ternary System.</a>
+        In this research, I observed changes in microstructures and mechanical properties of (W,Ti,Nb)C-Co Ternary system and also understood the effects of NbC addition on the microstructure of a ternary system. I achieved some good results, showing how to increase both toughness and hardness at the same time by controlling the WC grain growth.
```

Comment:
with things that you have done it is a good idea to create a projects button or tab rather then keeping them in your body